.circleci/config.yml

```diff
@@ -176,7 +176,7 @@ jobs:
                         --data '{ "branch": "main", "parameters": { "run_development_workflow": false, '\""$DEPLOYMENT_ENVIRONMENT"\"': true } }' ;
   
   run-ci-tests:
-        parallelism: 27 # Needs to be the same as the number of feature files
+        parallelism: 3 # Needs to be the same as the number of feature files
```

Comment:
But if the number is higher than no. feature files, it just wouldn't make use of excess parallelism right, or no?

Reply:
It does run 27 different containers on circleci, which then all run npm install, but only one of them runs the tests. Bit of a waste of resources. E.g. here - https://app.circleci.com/pipelines/github/LBHackney-IT/mtfh-tl-e2e-tests/4258/workflows/bf1fdee1-3b79-4db4-b0b5-a8e7c1ddfa14/jobs/10173/steps. All the containers ran, but only container 4 actually had any tests to run